src/Symfony/Component/VarDumper/Cloner/Data.php

```diff
@@ -270,12 +271,17 @@ public function dump(DumperInterface $dumper): void
         $cursor->hashType = -1;
         $cursor->attr = $this->context[SourceContextProvider::class] ?? [];
         $label = $this->context['label'] ?? '';
+        $options = $this->context['options'] ?? [];
 
         if ($cursor->attr || '' !== $label) {
             $dumper->dumpScalar($cursor, 'label', $label);
         }
         $cursor->hashType = 0;
         $this->dumpItem($dumper, $cursor, $refs, $this->data[$this->position][$this->key]);
+
+        if (false !== ($options['_trace'] ?? false) && $cursor->attr = $this->context[BacktraceContextProvider::class] ?? []) {
```

Comment:
comparisons to boolean... you missed testing this is `=== true` :P

Reply:
Actually, `_trace` can be either a bool or a integer. `false` disables the trace. If it's an integer, we limit the trace size. Also, the limit is disabled if `_trace` is <= 0. That's why we can't use `=== true` here :smile: But maybe limiting the stack trace isn't that useful and we may only accept a boolean for `_trace`?

Reply:
ah, makes sense then, let's keep as is :)